docs/OV_Runtime_UG/model_representation.md

```diff
@@ -132,7 +137,7 @@ The following code creates a model with several outputs:
 
 @endsphinxtabset
 
-## Model debug capabilities
+## Model Debug Capabilities
 
 OpenVINO™ provides several debug capabilities:
    - To receive additional messages about applied model modifications, rebuild the OpenVINO™ Runtime library with the `-DENABLE_OPENVINO_DEBUG=ON` option.
```

Comment:
There is a formatting issue from line 162 to 169 - they are be included in a code block in the output, which seems incorrect.